collect_app/src/main/res/xml/preferences.xml

```diff
@@ -75,6 +75,12 @@
             android:key="font_size"
             android:title="@string/font_size" />
         <CheckBoxPreference
+            android:id="@+id/instance_sync"
+            android:defaultValue="true"
+            android:key="instance_sync"
```

Comment:
I prefer creating Singleton classes for SharedPreferences. @nribeka @lognaturel Any thoughts?